tests/cover/test_type_lookup.py

```diff
@@ -28,7 +28,7 @@
 # Build a set of all types output by core strategies
 blacklist = [
     'builds', 'iterables', 'permutations', 'random_module', 'randoms',
-    'runner', 'sampled_from', 'streaming',
+    'runner', 'sampled_from', 'streaming', 'fspaths',
```

Comment:
I'm still not convinced that emulating Windows path types on Unix and vice versa is a good idea. I'd like to file that under "leave it for a later enhancement" and would like to see a real use case first.

I've added PurePath support though.